Doc/library/time.rst

```diff
@@ -708,6 +626,101 @@ The module defines the following functions and data items:
       ('EET', 'EEST')
 
 
+Clock ID Constants
+------------------
+
```

Comment:
I added one comment right under the constants heading.  It seemed that all the constants would apply to clock_getres and clock_getttime, so hopefully this is appropriate.